sdk/search/Azure.Search.Documents/src/SearchClientOptions.cs

```diff
@@ -35,13 +35,18 @@ public enum ServiceVersion
             /// The 2021_04_30_Preview version of the Azure Cognitive Search service.
             /// </summary>
             V2021_04_30_Preview = 2,
-            #pragma warning restore CA1707
+
+            /// <summary>
+            /// The 2023_07_01_Preview version of the Azure Cognitive Search service.
+            /// </summary>
+            V2023_07_01_Preview = 3,
```

Comment:
Should `V2021_04_30_Preview` be dropped

Reply:
Just for context, we typically do, but more often because there are breaking changes from preview to preview. If there's not, it could be left but still atypical.